src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} [%level] [%logger{1}] - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Logger name="life.qbic" level="info" additivity="false">
+            <AppenderRef ref="Console"/>
+        </Logger>
+        <Root level="all">
+            <AppenderRef ref="Console"/>
+        </Root>
+    </Loggers>
+</Configuration>
```

Comment:
File not in use anymore. Can be deleted.

Reply:
why is this no longer used?

Reply:
Currently the library does not log anything so the file is not needed. If Log4J should be added as a dependency again the file can be added again, too. If you want to use the library primarily in Liferay 7, for me it would make sense to get the logging directly from the Liferay kernel so that you have one less dependency to deploy.

Reply:
Thanks for the elaboration Jannis :)

Using Liferay's logging functionality does create a dependency upon the Liferay platform itself. I do not know what the library should achieve. Do we go for a 3-tier topology design? Should that be the case, this would probably contain business logic and should not depend on Liferay. 

For now, I guess we can delete the file as I also do not expect there to be logging here and the decision on this can be deferred until later or never :) :+1: